### PR TITLE
Docs for 4 Google AI connectors

### PR DESCRIPTION
**Pull request by zweedao**

### CODE REVIEW

**Comment by zweedao** [comment]
@graceliching doc for Google AI connectors. Can you help to review 👌

File: connectors/google-speech-to-text/action-short-speech.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+| Encoding | Encoding of the audio file. Note that Google has not supported MP3 input yet. MP3 file must be converted to one of [these supported encodings](https://cloud.google.com/speech-to-text/docs/reference/rest/v1/RecognitionConfig#AudioEncoding) before inputing into this action. |
+| Sample rate in Hertz | Sample rate of the audio file. Valid value range is 8000-48000. Usual optimal rate is 16,000, but varies across encodings and settings. Please refer to [Google documentation](https://cloud.google.com/speech-to-text/docs/reference/rest/v1/RecognitionConfig#audioencoding) to figure out the best sample rate. |
+| Enable word time offsets | If true, the output includes a list of words, the start and end time offsets (timestamps) for those words. |
+
+## Output fields
+
+| Field name | Description |
+|---|---|
+| Transcript | The extracted text transcript from the audio. |
+| Confidence score | Level of confidence Google puts in this result, with 0.0 being the worst and 1.0 being the best. |
```
Comment (graceliching) [inline connectors/google-speech-to-text/action-short-speech.md]: The confidence level for the entire transcription, with 0.0 as the lowest and 1.0 as the highest.

File: connectors/google-speech-to-text/action-short-speech.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+|---|---|
+| Audio content | Accept a content datapill, or audio content as a string. Workato will automatically convert the provided audio content into base-64 format. |
+| Language | The language code of the audio file, as listed [here](https://cloud.google.com/speech-to-text/docs/languages). |
+| Encoding | Encoding of the audio file. Note that Google has not supported MP3 input yet. MP3 file must be converted to one of [these supported encodings](https://cloud.google.com/speech-to-text/docs/reference/rest/v1/RecognitionConfig#AudioEncoding) before inputing into this action. |
+| Sample rate in Hertz | Sample rate of the audio file. Valid value range is 8000-48000. Usual optimal rate is 16,000, but varies across encodings and settings. Please refer to [Google documentation](https://cloud.google.com/speech-to-text/docs/reference/rest/v1/RecognitionConfig#audioencoding) to figure out the best sample rate. |
+| Enable word time offsets | If true, the output includes a list of words, the start and end time offsets (timestamps) for those words. |
+
+## Output fields
+
+| Field name | Description |
```
Comment (graceliching) [inline connectors/google-speech-to-text/action-short-speech.md]: For this table, it's unclear that start/end time and word are nested under the list of 'words'. Best to nest it in the table or provide a screenshot just above :)

File: connectors/google-text-to-speech.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+# Google Text-to-Speech
+[Google Text-to-Speech](https://cloud.google.com/text-to-speech/) allows converting text into natural-sounding speech with 30 voices, available in multiple languages and variants. It applies DeepMind’s research in WaveNet and Google’s neural networks to deliver high fidelity audio.
+
+Workato's Google Text-to-Speech connector allows you to build powerful automation and connect with other applications in the workflow.
+
+## API version
+The Google Text-to-Speech connector uses [Cloud Text-to-Speech API v1](https://cloud.google.com/text-to-speech/docs/reference/rest/).
+
+## How to connect to Google Text to Speech on Workato
+Google Text-to-Speech uses OAuth 2.0 for authentication. Follow the [connecting apps instructions](https://docs.workato.com/connections.html) and login to your Google account to connect.
```
Comment (graceliching) [inline connectors/google-text-to-speech.md]: Google Text-to-Speech uses OAuth 2.0 for authentication. When prompted, follow the on-screen [connection instructions](/connections.md) and provide the credentials to your Google account to authorize the connection.

File: connectors/google-speech-to-text.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+# Google Speech-to-Text
+[Google Speech-to-Text](https://cloud.google.com/speech-to-text/) allows converting audio to text by applying neural network models. It recognizes 120 languages and variants. You can enable voice command-and-control, transcribe audio from call centers, and more.
+
+Workato's Google Speech-to-Text connector allows you to build powerful automation and connect with other applications in the workflow.
+
+## API version
+The Google Speech-to-Text connector uses [Cloud Text-to-Speech API v1](https://cloud.google.com/text-to-speech/docs/reference/rest/).
+
+## How to connect to Google Text to Speech on Workato
+Google Speech-to-Text uses OAuth 2.0 for authentication. Follow the [connecting apps instructions](https://docs.workato.com/connections.html) and login to your Google account to connect.
```
Comment (graceliching) [inline connectors/google-speech-to-text.md]: Google Speech-to-Text uses OAuth 2.0 for authentication. When prompted, follow the on-screen [connection instructions](/connections.md) and provide the credentials to your Google account to authorize the connection.

File: connectors/google-text-to-speech/action-synthesize-text.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+| Text | The text to convert into voice audio. Can accept a datapill or raw text. |
+| Voice language | The language code of the voice, as listed [here](https://cloud.google.com/speech-to-text/docs/languages). Note that the conversion accuracy differs across lanuguages. |
+| Voice gender | The gender of the voice. |
+| Audio encoding | The encoding of the output audio file. |
+
+## Output fields
+
+| Field name | Description |
+|---|---|
+| Audio content | String content of the converted audio. |
```
Comment (graceliching) [inline connectors/google-text-to-speech/action-synthesize-text.md]: Does the user need to use the pill in a download file action? How do they obtain the audio from the string pill?

File: connectors/google-translate.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+# Google Translate
+[Google Translate](https://cloud.google.com/translate/) allows quickly translating an arbitrary text into any supported language using state-of-the-art Neural Machine Translation. Language detection is also available in case the source language is unknown. The underlying technology is updated constantly to include improvements from Google research teams, which results in better translations and new languages and language pairs.
+
+Workato's Google Translate connector allows you to build powerful automation and connect with other applications in the workflow.
+
+## API version
+The Google Translate connector uses [Google Translation API v2](https://cloud.google.com/translate/docs/reference/rest).
+
+## How to connect to Google Translate on Workato
+Google Translate uses OAuth 2.0 for authentication. Follow the [connecting apps instructions](https://docs.workato.com/connections.html) and login to your Google account to connect.
```
Comment (graceliching) [inline connectors/google-translate.md]: same as 2 above ^

File: connectors/google-vision/action-read-text.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+- **A scanned document:** Google Vision will return the extracted text, together with page, block, paragraph, word, and break information.
+
+  ![Scanned document](/assets/images/connectors/google-vision/document_text.png)
+
+## Input fields
+
+| Field name | Description |
+|---|---|
+| Image content | Can accept a file contents datapill, a string, a public URL or a Google Cloud Storage URI. |
```
Comment (graceliching) [inline connectors/google-vision/action-read-text.md]: Accepts a file contents data pill, a string, a public URL or a Google Cloud Storage URI.

**Comment by zweedao** [comment]
ok done @graceliching , addressed the comments.

**Comment by zweedao** [comment]
@graceliching can help to merge today?

**Pull request merged**